solr/core/src/java/org/apache/solr/cli/PostLogsTool.java

```diff
@@ -89,9 +90,16 @@ public void runImpl(CommandLine cli) throws Exception {
   }
 
   public void runCommand(String baseUrl, String root, String credentials) throws IOException {
+    if (URLUtil.isBaseUrl(baseUrl)) {
```

Comment:
this is interesting..  I wonder if "url" is a good name for this parameter?   I like the check...    --collection-url?  -collection-url..   though we dont have any other commands with a - in the name....   at least we have the check.